plaidml/bridge/openvino/ops/topk.cpp

```diff
@@ -0,0 +1,84 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "plaidml_ops.hpp"
+
+#include "ngraph/opsets/opset.hpp"
+#include "ngraph/opsets/opset4.hpp"
+
+#include "plaidml/op/op.h"
+
+using namespace plaidml;          // NOLINT[build/namespaces]
+using namespace InferenceEngine;  // NOLINT[build/namespaces]
+using Direction = edsl::SortDirection;
+using Mode = ngraph::opset4::TopK::Mode;
+using Sort = ngraph::opset4::TopK::SortType;
+
+namespace PlaidMLPlugin {
+
+void registerTopK() {
+  registerOp("topk", [](const Context& ctx) {
+    auto* layer = ngraph::as_type<ngraph::opset4::TopK>(ctx.layer);
+    IE_ASSERT(ctx.operands.size() == 2);
+    auto I = ctx.operands.at(0);
+    auto K = layer->get_k();
+    auto axis = layer->get_axis();
+    auto mode = layer->get_mode();
+    auto sort = layer->get_sort_type();
+    long ndims = I.rank();
+
+    Direction direction;
+    switch (mode) {
+      case Mode::MAX:
+        direction = Direction::DESC;
+        break;
+      case Mode::MIN:
+        direction = Direction::ASC;
+        break;
+      default:
+        THROW_IE_EXCEPTION << "invalid topk sort mode";
+    }
+
+    edsl::Tensor values, indices;
+    values = op::sort(I, axis, direction);
+    indices = edsl::argsort(I, axis, direction);
```

Comment:
nit: better to define a variable with it being initialized at construction.

```
edsl::Tensor values = op::sort(I, axis, direction);
edsl::Tensor indices = edsl::argsort(I, axis, direction);
```

Reply:
OK. It's done.